action.yml

```diff
@@ -2,6 +2,9 @@ name: "Crowdin Nested JSON Postprocessor"
 description: "Runs a Python script that remove empty string in JSON caused by Crowdin not respecting skip untranslated strings"
 
 inputs:
+  token:
+    description: "The GitHub personal access token required to push commits."
```

Comment:
#### 👋 Suggested change:

This removes the redundant description, there's no need to inform the user about the required input as this should be highlighted by the documentation, and/or their preferred code editor (like GitHub Web IDE).

```suggestion
    description: "GitHub's Personal Access Token"
```